src/lib/config/index.ts

```diff
@@ -58,11 +58,8 @@ export const BOOKING_CONFIRMED_NOTIFY_TEMPLATE_ID =
 	'8efb52bd-9ada-402e-8fab-84a751bf4a71'
 
 export const BOOKING_NOTIFY_RECIPIENTS = [
-	// 'david.acquet@cabinetoffice.gov.uk',
```

Comment:
Have these people decided not to be part of the gov notify project now?

Reply:
Not removed by this branch